lib/cocoapods/installer/analyzer.rb

```diff
@@ -354,6 +358,12 @@ def analyze_host_targets_in_podfile(aggregate_targets, embedded_aggregate_target
           end
         end
 
+        unless cli_host_with_frameworks.empty?
+          UI.warn "The Podfile contains command line tool target(s) (#{cli_host_with_frameworks.map(&:name).to_sentence}) which are attempting to integrate dynamic frameworks." \
```

Comment:
nit wrap the target names with  backticks

Reply:
kept it as this style to match https://github.com/CocoaPods/CocoaPods/pull/8767/files#diff-f5c3fee776e864ada6541431a1532b00R376